docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server might be different according to the platform developer.
+Therefore, this API just offers general functions for update, and platform developers must develop the application using update control API properly.
+
+>  **Note**
+>
+> As a default, the update agent works with [SmartThings Device Manager](https://console.smartthingsdm.com/) is available.
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API (in application), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/updatecontrol.admin</privilege>
+   </privileges>
+```
+
+2. To use the functions and data types of the update control API, include the `<update_control.h>` header file in your application:
+
+```cpp
+#include <update_control.h>
+```
+
+3. If some initialization is required to use update control functions, you can use `update_control_initialize()`:
+
+```cpp
```

Comment:
Please add indentation to this code block.
As this code block is the sub item of above, it need 4 spaces in front of the line from 43 to 49.
Also please remove "cpp".

Also, I'm not certain whether the `update_control_initialize()` and `update_control_deinitialize()` are Prerequisites.
"If some initialization is required .." -> Is this initialization optional?
How about moving these to "Connecting to Server" section as "Connecting to Server" is mentioning update_control_initialize()?

Reply:
@mijin85cho Thank you for the comment.
It looks better to move it to "Connecting to Server" section. Please confirm the latest patch.